spec/greenhouse_api/client_spec.rb

```diff
@@ -80,7 +80,7 @@
   end
 
   describe '#list_candidates' do
-    subject(:list_candidates) { client.list_candidates(params) }
+    subject(:list_candidates) { client.candidates.list_candidates(params) }
```

Comment:
Not sure if I'm calling this incorrectly - it keeps saying `NameError: uninitialized constant` when I run the specs and I can't figure out why

Reply:
Related to the circular reference comment above, I believe we need to require the new classes prior to using them.